pkg/processor/metricfrequencyprocessor/pdata.go

```diff
@@ -0,0 +1,24 @@
+package metricfrequencyprocessor
+
+import (
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"sort"
+)
+
+type PdataTimestampArr []pdata.Timestamp
```

Comment:
If you name this e.g. `PdataTimestampByValue` then your sort invocation will have more sense:

```go
sort.Sort(PdataTimestampByValue(timestamps))
```

Reply:
I'm not entirely sold, but changed